gapic-generator-java/src/main/java/com/google/api/generator/gapic/composer/ClientLibraryPackageInfoComposer.java

```diff
@@ -44,7 +48,10 @@ public class ClientLibraryPackageInfoComposer {
   private static final String SERVICE_DESCRIPTION_HEADER_PATTERN = "Service Description: %s";
 
   public static GapicPackageInfo generatePackageInfo(GapicContext context) {
-    Preconditions.checkState(!context.services().isEmpty(), "No services found to generate");
+    if (context.services().isEmpty()) {
```

Comment:
I think GapicContext has `containsServices()`. Probably can be `!context.containsServices()`?

Reply:
Good catch, I used the `containsServices` instead.